apps/dot_ext/oauth2_backends.py

```diff
@@ -36,15 +34,4 @@ def create_token_response(self, request):
                 fhir_body["patient"] = cw.fhir_id
                 body = json.dumps(fhir_body)
 
-            # When BENE chooses NOT to share demographic scopes, clean up previous access/refresh tokens.
-            app = token.application
-            user = token.user
-            scope = token.scope
-
-            # Does new token scope NOT contain BENE_PERSONAL_INFO_SCOPES?
-            if not set(settings.BENE_PERSONAL_INFO_SCOPES).issubset(scope.split()):
-                with transaction.atomic():
-                    AccessToken.objects.filter(application=app, user=user).exclude(id=token.id).delete()
-                    RefreshToken.objects.filter(application=app, user=user).exclude(access_token=token).delete()
-
```

Comment:
NOTE: Moved this functionality in to a utility function in apps/dot_ext/utils.py called `remove_application_user_pair_tokens(application, user)`.  Also, moved the call to this function closer to the beneficiary choice in the consent page form POST in apps/dot_ext/views/authorization.py `AuthorizationView.form_valid()`